docs/tutorials/cargo_tests/baked/build.rs

```diff
@@ -0,0 +1,35 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use icu_datagen::prelude::*;
+use std::path::PathBuf;
+
+fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
+
+    let out_dir = std::env::var_os("OUT_DIR").unwrap();
+    let mod_directory = PathBuf::from(out_dir).join("baked_data");
+    
+    // TODO(#3046): We shouldn't need to do this since it worked in 1.0
+    let _ = std::fs::remove_dir_all(&mod_directory);
+
+
+    icu_datagen::datagen(
+        Some(&[langid!("ru")]),
+        // Note: These are the keys required by `PluralRules::try_new_cardinal_unstable`
+        &[icu::plurals::provider::CardinalV1Marker::KEY],
+        &SourceData::default()
+            .with_cldr_for_tag(SourceData::LATEST_TESTED_CLDR_TAG, CldrLocaleSubset::Modern)
+            .expect("Source data should download successfully"),
```

Comment:
this doesn't do any downloading, so that message is not accurate

Reply:
I guess the function is infallible now? This is probably a leftover from when I wrote this file against 1.0. I added this to https://github.com/unicode-org/icu4x/issues/2856

Reply:
It is, thanks for adding